genomicFileHandler/concat.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import argparse, os, sys
+
+MY_DIR = os.path.dirname(os.path.realpath(__file__))
+sys.path.append( MY_DIR )
```

Comment:
Why this explicit append?

Reply:
I think one of my script was throwing out exceptions until I added this. Don't know why. I'll figure that out some other time.